examples/pxScene2d/src/Makefile

```diff
@@ -153,23 +176,40 @@ LDNODEV8 = -L$(NODEDIR)/out/Release
 #LDNODE   = -L$(NODEDIR)/out/Debug/obj.target
 #LDNODEV8 = -L$(NODEDIR)/out/Debug
 
+ifeq ($(EMBEDDED_BUILD),1)
```

Comment:
Please refer to the comments above.  We are adding a lot of new defines (such as EMBEDDED_BUILD) that are defined outside of the Makefile.  We should not introduce any new logic that depends on defines or variables that are not defined in this Makefile.